demos/browser/server.js

```diff
@@ -30,6 +30,8 @@ console.info('Using endpoint', endpoint);
 
 chime.endpoint = new AWS.Endpoint(endpoint);
 
+const INGESTION_URL = process.env.INGESTION_URL;
```

Comment:
Do we need this anymore?

Reply:
Thanks for the catch. Have removed it. It is not needed now.